rust/lance-encoding/src/format.rs

```diff
@@ -133,6 +133,12 @@ impl ProtobufUtils {
         }
     }
 
+    pub fn binary_miniblock() -> ArrayEncoding {
+        ArrayEncoding {
+            array_encoding: Some(ArrayEncodingEnum::BinaryMiniblock(BinaryMiniBlock {})),
```

Comment:
not important to this PR but I defined proto buffer 
```
message BinaryMiniBlock {
}
```
but here it is `BinaryMiniblock`
`B` -> `b`
don't know the reason.

Reply:
It is because you have `BinaryMiniBlock binary_miniblock = 15;` and not `BinaryMiniBlock binary_mini_block = 15;`

Reply:
oh! gotcha, thanks!

Reply:
fixed.